docs/source/installation.rst

```diff
@@ -71,6 +78,12 @@
     {{ app_name }} Python package
     *****************************
 
+    We recommend you `create a virtual Python environment <https://docs.python.org/3/tutorial/venv.html>`_ to work with MatFlow to separate versions for different projects and avoid dependency conflicts.
+    Check that you have the correct environment activated by confirming that the name of the environment appears in brackets in your prompt.
+
+    {{ app_name }} is compatible with Python 3.9, 3.10, 3.11, and 3.12.
```

Comment:
Is there a way to autogenerate the list of supported Python versions, so this doesn't get out of date?